src/molecules/formfields/CheckboxWrapper/checkbox_wrapper.module.scss

```diff
@@ -0,0 +1,26 @@
+@import '../shared/formfields.module.scss';
+
+.checkbox-list {
+  @extend %form-field;
+}
+
+.label {
+  border-bottom: 1px solid color('neutral-5');
+}
+
+.content {
+  padding: ru(.75);
+}
+
+
+@media #{$small-only} {
+  .content {
+    @include arrange-in-block-grid($per-row: 1, $gutter: ru(.75));
```

Comment:
I left some of the legacy stuff. Trying to do the thinnest slice to get this formfield broken up.